dynload.ml

```diff
@@ -0,0 +1,56 @@
+
+(* dynamically load plugins.*)
+
+module StrM = Map.Make(String)
+module StrS = Set.Make(String) 
+module StrMM = MoreLabels.Map.Make(String)
+module StrMS  = MoreLabels.Set.Make(String)
+
+
```

Comment:
Add a comment explaining that this is to reference all OCaml standard libraries so that they get included, hence we may need to modify this if the standard library changes.  (The OCaml -linkall flag might do this automagically instead, we should check.)